lib/unstable/Config/Exceptions/TypeConflictException.php

```diff
@@ -12,6 +12,8 @@
 
 /**
  * @experimental 31.0.0
+ * @deprecated use \OCP\Config\Exceptions\TypeConflictException
```

Comment:
Do we need to add the version to all of them as we do for OCP?

```suggestion
 * @deprecated 32.0.0 use \OCP\Config\Exceptions\TypeConflictException
```